scripts/motorInfo

```diff
@@ -24,118 +24,125 @@ fi
 while getopts "f:s:e:" OPTION
 do
     case $OPTION in
-    f) 
-	PVFIELDS=`echo $OPTARG | sed s/','/' '/g`
+    f)
+        PVFIELDS=${OPTARG//,/ }
 	;;
     s)
+        # shellcheck disable=SC2034
         STARTTIME=$OPTARG
 	;;
     e)
+        # shellcheck disable=SC2034
         ENDTIME=$OPTARG
 	;;
+    *)
+        echo "Unknown flag used"
+        usage
+        exit 1
     esac
 done
 
-shift $(($OPTIND - 1))
+shift $((OPTIND - 1))
 
 if [ $# -lt 2 ]; then
     usage
     exit
 fi
 
-MOTOR_PV=`echo $1 | cut -d . -f 1`
-if [[ $1 ==  $MOTOR_PV ]]; then
+MOTOR_PV=$(echo "$1" | cut -d . -f 1)
+if [[ $1 ==  "$MOTOR_PV" ]]; then
     FIELD=''
 else
-    echo 1: $1
-    echo MOTOR_PV $MOTOR_PV
-    FIELD=`echo $1 | cut -d . -f 2`
+    echo "1: $1"
+    echo "MOTOR_PV $MOTOR_PV"
+    # shellcheck disable=SC2034
+    FIELD=$(echo "$1" | cut -d . -f 2)
 fi
 
 source /reg/g/pcds/setup/pcds_shortcuts.sh
 
 if [[ $PVFIELDS == '' ]] && [[ "$2" != *pmgr* ]]; then
-    IOC_DIR='/reg/g/pcds/epics/'`find_pv $MOTOR_PV | grep DIR | grep ims | awk '{print $2}'`'/build/autosave'
-    PVLIST=`grep $MOTOR_PV $IOC_DIR/*req`
+    IOC_DIR='/reg/g/pcds/epics/'$(find_pv "$MOTOR_PV" | grep DIR | grep ims | awk '{print $2}')'/build/autosave'
+    PVLIST=$(grep "$MOTOR_PV" "$IOC_DIR"/*req)
     for pv in $PVLIST; do
-	PVFIELDS=`echo $pv | awk -F "$MOTOR_PV" '{ print $2 }'`' '$PVFIELDS
+	PVFIELDS=$(echo "$pv" | awk -F "$MOTOR_PV" '{ print $2 }')' '$PVFIELDS
     done
 fi
 
-if [ $2 == 'autosave' ]; then
-    IOC_DIR='/reg/d/iocData/'`find_pv $MOTOR_PV | grep IOC: | head -n 1 | awk '{print $2}'`'/autosave'
+if [[ $2 == 'autosave' ]]; then
+    IOC_DIR='/reg/d/iocData/'$(find_pv "$MOTOR_PV" | grep IOC: | head -n 1 | awk '{print $2}')'/autosave'
 
-    files=`ls -t $IOC_DIR/*sav_1*`
+    files=$(ls -t "$IOC_DIR"/*sav_1*)
```

Comment:
While you did not introduce this, I'm pretty sure this should be either *sav_* or *sav_2* to get more recent data. I'm leaning to just remove the 1: having the _ is supposed to skip the set of the currently rotation sav* files.